controllers/account.go

```diff
@@ -291,20 +293,36 @@ func (c *ApiController) GetUserinfo() {
 	c.ServeJSON()
 }
 
-// GetHumanCheck ...
+// GetCaptcha ...
 // @Tag Login API
-// @Title GetHumancheck
-// @router /api/get-human-check [get]
-func (c *ApiController) GetHumanCheck() {
-	c.Data["json"] = HumanCheck{Type: "none"}
-
-	provider := object.GetDefaultHumanCheckProvider()
-	if provider == nil {
+// @Title GetCaptcha
+// @router /api/get-captcha [get]
+func (c *ApiController) GetCaptcha() {
+	applicationId := c.Input().Get("applicationId")
+	isPreview := c.Input().Get("isPreview")
+	captchaProvider := &object.Provider{}
+	if isPreview == "true" {
+		provider, err := object.GetPreviewCaptchaProvider(applicationId)
```

Comment:
There should NOT be preview in `GetPreviewCaptchaProvider()`, we should not see any `preview` wording except the frontend page UI.